iree/turbine/kernel/_support/tracing.py

```diff
@@ -158,8 +158,6 @@ def get_subgraph(self, name: str) -> fx.Graph:
         return self.region_graph.subgraphs[name]
 
     def add_subgraph(self, name: str, graph: fx.Graph):
-        for _, subgraph in self.region_graph.subgraphs.items():
```

Comment:
So one question here, do we need lines 148-155 in __init__ above? If we need each graph to access all other subgraphs then I think we still need this, otherwise you can get rid of both?

Reply:
Will try remove and see if everything is still alive, looking at the commit that checked that in, we may not need that

Reply:
removed, needed to standardize search for subgraphs by using `get_root_graph` :)

Reply:
nice!